client/reader/follow-button/follow-sources.jsx

```diff
@@ -10,5 +10,7 @@ export const READER_FEED_SEARCH = 'reader-feed-search-result';
 export const READER_FOLLOWING_MANAGE_URL_INPUT = 'reader-following-manage-url-input';
 export const READER_FOLLOWING_MANAGE_SEARCH_RESULT = 'reader-following-manage-search-result';
 export const READER_FOLLOWING_MANAGE_RECOMMENDATION = 'reader-following-manage-recommendation';
+export const READER_FULL_POST = 'reader-full-post';
```

Comment:
As this file is now being used for more than just the regular follow button, I suggest we move it to `reader/follow_sources.js` in a followup PR.